bin/ci-mirror.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+# Build an image that renders and copies static content to an S3 bucket
+# Ping DMS when complete
+
+export IMAGE=mozmeao/lumbergh-deploy:`git rev-parse --short HEAD`
+docker build -f ./bin/Dockerfile . -t ${IMAGE}
+docker run \
+      -e AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}" \
```

Comment:
NIT: If the env var is in the environment, you can just `-e AWS_ACCESS_KEY_ID` and [it'll use the value of that var in the env](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file). Might make things easier for next time.

Reply:
nice, I'll use that syntax.